README.md

```diff
@@ -1,2 +1,46 @@
 # react-page-swapper
-An orchestrator that eases out the implementation of page transitions
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/react-page-swapper
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-page-swapper.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-page-swapper.svg
+[build-status-url]:https://github.com/moxystudio/react-page-swapper/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-page-swapper/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-page-swapper
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-page-swapper/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-page-swapper
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-page-swapper.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-page-swapper?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-page-swapper.svg
+
+An orchestrator that eases out the implementation of page transitions.
+
+## Installation
+
+```sh
+$ npm install @moxy/react-page-swapper
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+## Motivation
+
+
+## Demo
+
+You may see a simple demo of `@moxy/react-page-swapper` at [https://moxystudio.github.io/react-page-swapper](https://moxystudio.github.io/react-page-swapper/).
+
+## Usage
+
+
+## Tests
+
+```sh
+$ npm test
+$ npm test -- --watch # during development
+```
+
+## License
+
+Released under the [MIT License](https://www.opensource.org/licenses/mit-license.php).
```

Comment:
Shouldn't you do `[MIT License](./LICENSE)` ?

Reply:
Well maybe. All our repos are like this though. 🤔

Reply:
Resolving this as it's a non-issue.